### PR TITLE
Prepare for publishing to npm

### PR DESCRIPTION
**Pull request by tadatuta**
Update deps, provide ignores

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bem/bem-history.git"
+  },
+  "homepage": "https://bem.info/libs/bem-history/",
+  "main": "",
+  "keywords": [
+    "bem",
+    "history",
+    "history api"
```
Comment (mishanga) [inline bower.json]: +window location

File: bower.json [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
+  ],
   "ignore": [
-    ".bem/cache",
-    "node_modules",
-    "libs",
-    "*.bundles"
+    "*",
+    "!common.blocks",
```
Comment (mishanga) [inline bower.json:18]: `"!*.blocks",`

File: bower.json [hunk 3]
```diff
@@ -1,19 +1,30 @@
...
+    "!desktop.blocks",
+    "!touch.blocks",
+    "!*.md",
+    "!LICENSE.txt"
```
Comment (mishanga) [inline bower.json:22]: Можно сразу добавить этот файл, его сейчас нет.

**Comment by mishanga** [comment]
В остальном ок.

File: .npmignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+*.blocks/*/*.tests
+*.blocks/*/*.tmpl-specs
+*.blocks/*/*.spec.js
```
Comment (remnev) [inline .npmignore:3]: FYI https://github.com/npm/npm/wiki/Files-and-Ignores#details-1

> Entries matched by files will be included, regardless of .npmignore settings. This is [also a known issue](https://github.com/npm/npm/wiki/Files-and-Ignores#known-issues).

So this file doesn't bring a value.

Comment (tadatuta) [reply .npmignore:3]: Looks like we can just wait for a fix on npm side as they are already in progress.

**Pull request merged**